lib/redux/constants/errors.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
   E_HID_CONNECTION: 'Hmm, your device doesn\'t appear to be connected',
   E_INIT: 'Error during app initialization',
   E_MEDTRONIC_UNSUPPORTED: 'Tidepool does not support Minimed pumps 522, 722 or older, or the newer 6-series pumps. Sorry...',
+  E_MEDTRONIC_UPLOAD: 'Is there another device (e.g. CareLink/OpenAPS/Loop) talking to your pump? If not, see error details below or contact Tidepool Support.',
```

Comment:
I agree, but there's not really enough space at the moment. Once we have an updated design (see https://trello.com/c/8nJtBvT7) we should consider updating this error message.